feature/floormap/src/main/java/io/github/droidkaigi/confsched2019/floormap/ui/FloorMapFragment.kt

```diff
@@ -61,6 +66,39 @@ class FloorMapFragment : DaggerFragment() {
         }
         floorMapActionCreator.load()
     }
+
+    private fun setupFloorMapPager() {
+        binding.floorMapTabLayout.setupWithViewPager(binding.floorMapViewPager)
+        binding.floorMapViewPager.adapter = object : PagerAdapter() {
+            override fun instantiateItem(container: ViewGroup, position: Int): Any {
+                val itemBinding = DataBindingUtil.inflate<ItemFloorMapBinding>(
+                    LayoutInflater.from(context),
+                    R.layout.item_floor_map,
+                    container,
+                    true
+                )
+                val resId = resources.getIdentifier(
```

Comment:
Thanks! 
I understand your feelings. But it is a little danger. because That image is shrunk.
So how about use `when` here.

val resId = when(floorList[position]) {
...

Reply:
Thanks for your review! You're right!!
I will fix it like that.